charts/frontend/src/main/scala/io/udash/wrappers/highcharts/config/tooltip/Tooltip.scala

```diff
@@ -0,0 +1,405 @@
+    */
+  val split: js.UndefOr[Boolean] = js.undefined
+
+  /**
+    * CSS styles for the tooltip. The tooltip can also be styled through the CSS
+    * class <code>.highcharts-tooltip</code>.
+    *
+    * @example <a href="http://jsfiddle.net/gh/get/jquery/3.1.1/highcharts/highcharts/tree/master/samples/highcharts/tooltip/style/" target="_blank">Greater padding, bold text</a>
+    */
+  val style: js.UndefOr[js.Object] = js.undefined
+
+  /**
+    * Use HTML to render the contents of the tooltip instead of SVG. Using HTML allows advanced formatting like tables and images in the tooltip.
+    * It is also recommended for rtl languages as it works around rtl bugs in early Firefox.
+    *
+    * @example <a href="http://jsfiddle.net/gh/get/jquery/3.1.1/highcharts/highcharts/tree/master/samples/highcharts/tooltip/footerformat/" target="_blank">A table for value alignment</a>. <a href="http://jsfiddle.net/gh/get/jquery/3.1.1/highcharts/highcharts/tree/master/samples/highcharts/tooltip/fullhtml/" target="_blank">Full HTML tooltip</a>.
+    */
+  val useHTML: js.UndefOr[Boolean] = js.undefined
+
+  /**
+    * How many decimals to show in each series' y value. This is overridable in each series' tooltip options object. The default is to preserve all decimals.
+    *
+    * @example <a href="http://jsfiddle.net/gh/get/jquery/3.1.1/highcharts/highcharts/tree/master/samples/highcharts/tooltip/valuedecimals/" target="_blank">Set decimals, prefix and suffix for the value</a>
+    */
+  val valueDecimals: js.UndefOr[Double] = js.undefined
+
+  /**
+    * A string to prepend to each series' y value. Overridable in each series' tooltip options object.
+    *
+    * @example <a href="http://jsfiddle.net/gh/get/jquery/3.1.1/highcharts/highcharts/tree/master/samples/highcharts/tooltip/valuedecimals/" target="_blank">Set decimals, prefix and suffix for the value</a>
+    */
+  val valuePrefix: js.UndefOr[String] = js.undefined
+
+  /**
+    * A string to append to each series' y value. Overridable in each series' tooltip options object.
+    *
+    * @example <a href="http://jsfiddle.net/gh/get/jquery/3.1.1/highcharts/highcharts/tree/master/samples/highcharts/tooltip/valuedecimals/" target="_blank">Set decimals, prefix and suffix for the value</a>
+    */
+  val valueSuffix: js.UndefOr[String] = js.undefined
+
+  /**
+    * The format for the date in the tooltip header if the X axis is a datetime axis.
+    * The default is a best guess based on the smallest distance between points in the chart.
+    *
+    * @example <a href="http://jsfiddle.net/gh/get/jquery/3.1.1/highcharts/highcharts/tree/master/samples/highcharts/tooltip/xdateformat/" target="_blank">A different format</a>
+    */
+  val xDateFormat: js.UndefOr[String] = js.undefined
+}
+
+object Tooltip {
+  import scala.scalajs.js.JSConverters._
+
+  @js.native
+  trait FormatterData extends js.Object {
+    val percentage: Double = js.native
+    val point: Point = js.native
```

Comment:
I'm afraid one of {`point`, `points`} is undefined depending on the plurality, just Highcharts things...